.ansible-lint

```diff
@@ -0,0 +1,14 @@
+---
+skip_list:
+  - '106'
```

Comment:
Worth a comment to explain what this is? Had to look it up.

Reply:
Ah yeah, came from Aaron's suggested config but I'll remove as no failures.